erpnext_documentation/www/docs/user/manual/en/regional/india/lower-deduction-certificate.md

```diff
@@ -0,0 +1,40 @@
+<!-- add-breadcrumbs -->
+# Lower Deduction Certificate
+
+According to tax withholding category, a person responsible for making payments is required to deduct tax at source at prescribed rates. Instead of receiving tax on your income from you at a later date, the govt wants the payers to deduct tax beforehand and deposit it with the government. However this scheme of deducting tax at source may cause hardship to few taxpayers who may not have taxable income for that fiscal year.
```

Comment:
```suggestion
According to the tax withholding category, a person responsible for making payments is required to deduct tax at source at prescribed rates. Instead of receiving tax on your income from you at a later date, the government wants the payers to deduct tax beforehand and deposit it with them. However, this scheme of deducting tax at source may cause issues to a few taxpayers who may not have taxable income for that fiscal year.
```